docs/gettingstarted/autoupdate.md

```diff
@@ -0,0 +1,42 @@
+Forge Update Checker
+====================
+
+Forge provides a very lightweight opt-in update-checking framework. All it does is check for updates, then show a flashing icon on the Mods button of the main menu and mod list if any mods have an available update, along with the respective changelogs. It *does not* download updates automatically.
+
+Getting Started
+---------------
+
+The first thing you want to do is specify the `updateJSON` parameter in your `@Mod` annotation. The value of this parameter should be a valid URL pointing to an update JSON file. This file can be hosted on your own web server, or on GitHub, or wherever you want, as long as it can be reliably reached by all users of your mod.
```

Comment:
Does it matter in this case? 
also their website says you get throttled if it happens too much, big mods with lots of players starting up the mod every time can get their rawgit slammed